bigbluebutton-html5/imports/ui/components/user-list/component.jsx

```diff
@@ -1,26 +1,34 @@
-import React, { Component } from 'react';
-import styles from './styles.scss';
-import { Link } from 'react-router';
+import React, { Component, PropTypes } from 'react';
 import { withRouter } from 'react-router';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import styles from './styles.scss';
 import { FormattedMessage } from 'react-intl';
+import classNames from 'classnames/bind';
```

Comment:
As far as I can see you don't need `bind` here since the values of your className{} and the names of the classes you put in are exactly the same.